save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -535,7 +527,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                     }
                     img {
                         className = ClassName("avatar avatar-user width-full border color-bg-default rounded-circle")
-                        src = state.image?.path?.let {
+                        src = state.organization?.avatar?.let {
```

Comment:
does it mean that we should now re-load `state.organization` after uploading an image?
previously we set `image` and used it to take an url

Reply:
or window.location.reload() does it?

Reply:
Yes. Added reload() for it